doc-Monitoring_Alerts_and_Reporting/topics/Integrating_Prometheus_Alerts.adoc

```diff
@@ -0,0 +1,130 @@
+[[integrating_prometheus_alerts]]
+== OpenShift Prometheus Alerts
+
+[NOTE]
+====
+This feature is available with this release of {product-title} as a technology preview only. For more information on the support scope for features marked as technology previews, see https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope].
+====
+
+Prometheus is used as an external alerting component. {product-title_short} integrates Prometheus alerts for use with OpenShift Container Platform. {product-title_short} collects EmsEvents from Prometheus, generates alerts based on these events, and then attaches alerts to inventory objects.
```

Comment:
@moolitayer Wanted to check if this intro text correctly captures the current feature and sequence of steps that is useful to the user? Any feedback here?

Reply:
Looks good. replace `EmsEvents` with `event` I think it's too technical for docs

Reply:
Fixed, thanks.